creator-node/src/apiHelpers.js

```diff
@@ -65,12 +65,9 @@ module.exports.handleResponseWithHeartbeat = (func) => {
 }
 
 const sendResponse = module.exports.sendResponse = (req, res, resp) => {
-  const endTime = process.hrtime(req.startTime)
-  const duration = Math.round(endTime[0] * 1e3 + endTime[1] * 1e-6)
-  let logger = req.logger.child({
-    statusCode: resp.statusCode,
-    duration
-  })
+  let duration = getDuration(req, resp)
```

Comment:
the function defined in src/logging.js only takes in 1 params - 2 are passed in here.
I think we can remove the resp second param - same for below use